packages/babel-runtime-corejs3/helpers/esm/possibleConstructorReturn.js

```diff
@@ -1,4 +1,4 @@
-import _typeof from "./typeof.js";
+import _typeof from "@babel/runtime-corejs3/helpers/typeof";
```

Comment:
Are you using windows? I just realized that this unwanted change (I'll revert it again) might be a bug in out build process.

Reply:
yes, i am using windows! I have reverted it just now!

Reply:
now i am rerunning `make bootstrap`